settings.py

```diff
@@ -2094,7 +2095,7 @@
     'icds-test': 'custom.icds_reports',
     'icds-cas': 'custom.icds_reports',
     'icds-dashboard-qa': 'custom.icds_reports',
-    'reach-test': 'custom.icds_reports',
+    'reach-test': 'custom.reach',
```

Comment:
@emord probably we need to add a possibility to enable two custom modules for one domain?

Reply:
I think that in the future we may need that. Currently they will be used in separate domains so I do not believe it will be necessary now.